packages/flutter_android_lifecycle/example/android/app/src/main/java/com/example/flutter_android_lifecycle_example/MainActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.example.flutter_android_lifecycle_example;
+
+import android.os.Bundle;
+import io.flutter.app.FlutterActivity;
+import io.flutter.plugins.GeneratedPluginRegistrant;
+
+public class MainActivity extends FlutterActivity {
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
```

Comment:
If we make this sample actually use `getLifecycle` we should get an e2e easy to write e2e test (just confirm that the app loaded without crashing)

Reply:
I added a test plugin to verify we can obtain the real Lifecycle.